### PR TITLE
Envest/update readme

### LINKED ISSUE #99: Major home page documentation update
There will be several necessary updates to `README.md`, including

- [x] Add information about options to shell scripts (namely: cancer type, predictor, ncores)
- [x] Update data sets, normalization methods, and predictor information
- [x] Update diagrams
- [ ] Update shared data links
- [x] Check all existing links
- [x] Mention `search_geo_arrayexpress.py`

### PR DESCRIPTION
**Pull request by envest**
Closes #99 

This PR seeks to make major updates to the README.md file ahead of manuscript resubmission.

I've tried to make changes consistent with our updated manuscript methods and data, including uploading new overview diagrams. Now that our bash scripts have command line arguments, I included those details. There is also now a table with links to each previous version and associated materials.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,77 @@
...
+3. Machine learning applications
+
+  - _Supervised learning_: 
+Three classifiers - LASSO, linear SVM, and Random Forest — were trained on each training set and tested on the microarray and RNA-seq holdout sets.
+The models were trained to predict tumor subtype (5 classes), and the binary mutation status of _TP53_ and _PIK3CA_.
+
+  - _Unsupervised learning_: 
+Holdout sets were projected onto and back out of the training set space using Principal Components Analysis to obtain reconstructed holdout sets.
+The trained subtype classifiers were used to predict on the reconstructed holdout sets.
+[PLIER](https://github.com/wgmao/PLIER) (pathway-level information extractor) identified active gene expression pathways in each cancer type.
```
Comment (jaclyn-taroni) [inline README.md:62]:
```suggestion
[PLIER](https://github.com/wgmao/PLIER) (Pathway-Level Information ExtractoR) identified active gene expression pathways in each cancer type.
```

File: README.md [hunk 2]
```diff
@@ -1,69 +1,77 @@
...
+Data from 150 glioblastoma (GBM) patients is available from the [Genomic Data Commons PanCan Atlas](https://gdc.cancer.gov/about-data/publications/pancanatlas).
+
 ```
-# To download data, run in top directory:
-sh brca_data_download.sh
+# To download data, run the data download script in the top directory:
+bash download_TCGA_data.sh
 ```
 
 ## Analysis
```
Comment (jaclyn-taroni) [inline README.md:31]: If you take my suggestion about moving the recreation instructions, I might change this to:

```suggestion
## Methods
```

File: README.md [hunk 3]
```diff
@@ -1,69 +1,77 @@
...
-Rscript run_experiments.R
+bash run_machine_learning_experiments.sh [cancer type] [prediction task] [n cores]
 ```
 
+where 
+
+- `[cancer type]` is `BRCA` or `GBM`
+- `[prediction task]` is `subtype`, `TP53`, or `PIK3CA`
+- `[n cores]` is the number of cores you want to run in parallel
+
```
Comment (jaclyn-taroni) [inline README.md:74]: Break this and running differential expression analysis into its own section, e.g., "Running individual experiments" below the Methods (or Analysis) section.

Comment (jaclyn-taroni) [reply README.md:74]: I think you could tuck the contents of "Other scripts" under the "Running individual experiments" header

File: README.md [hunk 4]
```diff
@@ -72,42 +80,57 @@ Here's a schematic overview of our main differential expression experiment:
...
 
+where 
+
+- `[cancer type]` is `BRCA` or `GBM`
+- `[subtype vs others]` is the subtype to be compared against all other subtypes
+- `[subtype vs subtype]` are the two subtypes to be compared (comma-separated, e.g. Her2,LumA)
+- `[subtype vs subtype small]` are the two subtypes to be compared at small sample sizes (comma-separated, e.g. Her2,LumA)
+- `[n cores]` is the number of cores you want to run in parallel
+
 ## Requirements
```
Comment (jaclyn-taroni) [inline README.md:110]: Move requirements way up to before downloading or at least before recreating the manuscript results.

File: README.md [hunk 5]
```diff
@@ -72,42 +80,57 @@ Here's a schematic overview of our main differential expression experiment:
...
 
-This analysis was performed in R. It requires R & Bioconductor packages 
-detailed in `check_installs.R` to be installed.
+We recommend using the docker image `envest/rnaseq_titration_results:R-4.1.2` to handle package and dependency installation.
+See `docker/R-4.1.2/Dockerfile` for more information.
 
-One github package (`TDM`) is required. To install, run:
+Our analysis (v2.0) was run using 7 cores on an AWS instance with 16 cores, 128 GB memory, and an allocated 1 TB of space.
```
Comment (jaclyn-taroni) [inline README.md:115]: How do you anticipate folks will interact with the container? Might want to add something like the `run` command here: https://github.com/AlexsLemonade/refinebio-examples/blob/staging/CONTRIBUTING.md#setting-up-the-docker-container

Comment (jaclyn-taroni) [reply README.md:115]: I would also consider putting the Docker documentation under "Recreate manuscript results" with its own subheader.

**Review by jaclyn-taroni — COMMENTED** [review]
Returning some high-level review comments! I've mostly focused on overall organization. I did not read the updates to the analysis section or the documentation of the different bash arguments closely yet.

**Comment by envest** [comment]
Thanks for high level review @jaclyn-taroni ! I've made all those changes and added some more links. Ready for 👀

**Review by jaclyn-taroni — COMMENTED** [review]
Also had some comments to return that are not captured in #120

**Review by jaclyn-taroni — APPROVED** [review]
🚀

**Pull request merged**